code/controllers/subsystem/persistence.dm

```diff
@@ -152,6 +155,15 @@ SUBSYSTEM_DEF(persistence)
 		return
 	saved_modes = json["data"]
 
+/datum/controller/subsystem/persistence/proc/LoadAntagReputation()
+	var/json_file = file("data/AntagReputation.json")
```

Comment:
Use a define for this file

Reply:
Similarly, the other things in persistence do not use defines. Should they _all_ be changed?

Reply:
just because we did things poorly in the past should not mean we shouldn't do them right now.

You don't need to do all the rest, just your own, unless you specifically want to.